userguide/subscription/includes/catalog-introduction.adoc

```diff
@@ -1,78 +1,76 @@
 * **Plan Change Timing** - Should a customer downgrading to a lower (paid) plan be active immediately or should the system wait for the next renewal date?
 
 
-The catalog is also used by the invoicing system to make sure each each `Customer` is billed at the right time and for the right amount. Information about **how** the customer is billed and **when** the customer is billed is also defined in the catalog using alignments and their associated rules.
+The catalog is also used by the invoicing system to make sure each `Customer` is billed at the right time and for the right amount. Information about **how** the customer is billed and **when** the customer is billed is also defined in the catalog using alignments and their associated rules.
 
-Catalog configuration allows to define if different subscriptions attached to a customer should be invoiced at the same time, or at the opposite if they should be billed separately on their own schedule (or a mix of both where some are grouped together on the same invoice and others are invoiced independently). In the simplest case, where all subscriptions are grouped together and there is a `BCD` defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that (`BCD`) day. Note that grouping subscriptions together may by default lead to some leading pro-ration when the subscription is started on a different date than the `BCD` (as the system needs to realign invoicing on that particular date). This behavior is configured in the catalog through the http://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules[Billing Alignment Rules].
+Catalog configuration allows the Kill Bill Administrator to define whether all subscriptions attached to a customer should be invoiced at the same time, or if they should be billed separately on their own schedule. There could also be a mix of both where some are grouped together on the same invoice and others are invoiced independently. In the simplest case, where all subscriptions are grouped together and there is a Bill Cycle Day (BCD) defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that day. Note that grouping subscriptions together may by default lead to some proration when the subscription is started on a different date than the BCD, as the system needs to realign invoicing on that particular date. This behavior is configured in the catalog through the http://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules[Billing Alignment Rules].
 
 ==== Creating the Catalog
 
-In this section we look at getting the top level catalog created and configured.
+A Kill Bill catalog, as we know, is an XML document. This section describes how to create a catalog configured to your particular needs.
```

Comment:
This is actually not necessarily true anymore. In the vast majority of cases, it is indeed an XML document.

For complex billing requirements, it can be defined in Java code instead (http://docs.killbill.io/latest/catalog_plugin.html).

For very simple requirements, we also have a GUI in Kaui to create _simple plans_ (see http://docs.killbill.io/latest/getting_started.html#_modifying_the_catalog).

It would be great if this could be somehow explained in this doc. The same concepts apply regardless of the configuration method though, and it's easiest to explain it via the XML method, so the rest of this doc could still focus only on the XML method.

Reply:
I have added some new text for this